CONTRIBUTING.adoc

```diff
@@ -16,6 +16,7 @@ This includes a variety of useful tools:
 * Agda with the right libraries set up to build our Agda code
 * `stylish-haskell`
 * `haskell-language-server`
+* `purescript-language-server`
```

Comment:
I was able to use the language-server + IDE without having to add it to nix-shell 🤔 ... But I don't think this hurts either way.

The only caveat is that you need to open as the root folder each purescript project, which is mitigated by the fact that you can have multiple projects in the same workspace

Reply:
What editor do you use? I think the VS Code plugin automatically downloads and installs purescript-language-server for you, so you might have it installed in your global environment

Reply:
ah, I do use VS Code, maybe later we can clarify this in the docs, if you use vscode is not necessary to launch the language-server manually